Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -87,14 +87,6 @@ protected void onCreate(Bundle savedInstanceState) {
         dropInViewModel = new ViewModelProvider(this).get(DropInViewModel.class);
         fragmentContainerView = findViewById(R.id.fragment_container_view);
 
-        dropInClient.getSupportedPaymentMethods(this, (paymentMethods, error) -> {
```

Comment:
Is this handled elsewhere and it was just duplicated here? or what?

Reply:
This was an optimistic prefetch. It still happens [here](https://github.com/braintree/braintree-android-drop-in/blob/f9ad3aca587509a1138c73afbdd80d8bd7970dbd/Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java#L255) when the bottom sheet is presented.

We may add a `prefetch()` or `warmup()` method on `DropInClient` to allow for explicit prefetching in the future, but at the moment we've punted on supported / vaulted payment method caching for a post GA release.